liferay-workspace/modules/blade.modellistener/README.markdown

```diff
@@ -0,0 +1,51 @@
+# Model Listener
+
+## What does this sample do when it's deployed?
+
+The Model Listener sample demonstrates adding a custom model listener to a Liferay Portal's out of the box entity. When deploying this sample with no customizations, a custom model listener will be added to `layouts`, listening for `onBeforeCreate` events. This means that any page creation will trigger this listener, which will execute before the new page being created.
```

Comment:
Hey @codyhoag, my goal there was to explain that it's adding this model listener to the Liferay portal's layout entity, which is an OOTB entity like user, group, etc.

I'm not sure if the new version is clear enough, as it can be understood as Layout the plugin, and we are talking about Layout the entity. Does it makes sense?

Just to double-check, did you publish this doc already? I was not able to find it in the main repo.

Let me know if you need anything else.